docs/index.md

```diff
@@ -5,13 +5,17 @@ title: RAVE AddIn
 Welcome to the Riverscapes Analysis Visualization Explorer (RAVE) software website. Here you can find general information about the RAVE software, including information on [opening projects](open-project.html), [symbolizing layers](symbology.html) and controlling how projects are displayed using [business logic XML](business-logic.html).
 
 <div style="text-align:center">
+<a class="button large" href="http://rave.riverscapes.xyz/install.html#qgis-plug-in">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Installation Instructions for QRAVE PlugIn for QGIS</a>
 <a class="button large" href="https://github.com/Riverscapes/RaveAddIn/releases/latest">
         <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
-        &nbsp;&nbsp;Download RAVE</a>
+        &nbsp;&nbsp;Download ArcRAVE AddIn for ArcGIS 10.X</a>
+<a class="button large" href="https://data.riverscapes.xyz">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Explore WebRAVE in Warehouse/a>
```

Comment:
open anchor tag with `Warehouse / a>` on 16. Needs to be `Warehouse</a>`. @shelbysawyer this is why website now looks like this:
![image](https://user-images.githubusercontent.com/9723905/129115732-2ae315e5-8ed5-4390-8782-633ce857c5ae.png)

@shelbysawyer are you testing these [MD sites on a local server](https://riverscapes.xyz/Tools/Technical_Reference/Documentation_Standards/WebSites/testing_locally.html)? If you were, you'd see this problem.

Reply:
@joewheaton No, I wasn't aware of how to do that. Thanks for the link.